### PR TITLE
Adding script to pull vaccine phases in from spreadsheet

### PR DESCRIPTION
**Pull request by ghop02**
Adds in a script to refresh the CMS data with data from the spreadsheet.  In the future I'd love to automate this a bit more but it works as a cli command for now

### CODE REVIEW

File: scripts/vaccination_alerts/parse-vaccination-phases.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  console.log(
+    `In CMS but not spreadsheet: ${inCmsNotSpreadsheet.map(
+      state => state.locationName,
+    )}`,
+  );
+
+  if (dryRun) {
+    console.log('Dry run -- exiting');
+    return;
+  }
```
Comment (pnavarrc) [inline scripts/vaccination_alerts/parse-vaccination-phases.ts:140]: I'm not sure if we need the dry run mode, the user can always discard changes to the CMS locally before committing them

File: scripts/vaccination_alerts/parse-vaccination-phases.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+      {
+        regions: sortBy(
+          [...updatedMatchingRecords, ...newStates, ...inCmsNotSpreadsheet],
+          state => state.locationName,
+        ),
+      },
+      null,
+      2,
+    ),
+  );
```
Comment (pnavarrc) [inline scripts/vaccination_alerts/parse-vaccination-phases.ts:154]: Consider creating a variable for the data item so we can check the type before writing to the file.

File: scripts/vaccination_alerts/parse-vaccination-phases.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    state: row['State'],
+    eligibilityUrl: row['Eligibility Information URL'],
+    currentlyEligible: row['Currently Eligible'].trimRight() !== '',
+    startDate: row['Estimated Start Date'],
+    phase: row['Phase'],
+    tier: row['Tier'],
+    groupDetails: row['Detailed Group'],
+    generalGroup: row['General Group'],
+    descriptionUrl: row['Expanded description url'],
+    notes: row['Notes for Internal QA'],
```
Comment (pnavarrc) [inline scripts/vaccination_alerts/parse-vaccination-phases.ts:52]: Can we use indices instead to make this more resilient?

**Review by pnavarrc — APPROVED** [review]
Looks good!

File: scripts/vaccination_alerts/parse-vaccination-phases.ts (hunk 3 above)
Comment (ghop02) [reply scripts/vaccination_alerts/parse-vaccination-phases.ts:52]: hmm I'm not sure that will really make it more resilient - very possible that a column is added and breaks it.  with indices I'm more concerned the failure will be silent

**Pull request merged**